addOns/dev/src/main/java/org/zaproxy/addon/dev/TestDirectory.java

```diff
@@ -110,14 +105,20 @@ public void handleMessage(HttpMessageHandlerContext ctx, HttpMessage msg) {
             }
 
             if (body == null) {
-                LOGGER.debug("Failed to find tutorial file {}", name);
-                body = server.getTextFile("404.html");
-                msg.setResponseBody(body);
-                msg.setResponseHeader(
-                        TestProxyServer.getDefaultResponseHeader(
-                                TestProxyServer.STATUS_NOT_FOUND,
-                                "text/html",
-                                msg.getResponseBody().length()));
+                if (name.equals(this.getName())) {
+                    // Redirect with a trailing slash
+                    getServer().redirect(name + "/", msg);
+                    return;
+                } else {
+                    LOGGER.debug("Failed to find tutorial file {}", name);
```

Comment:
Tutorial?

Reply:
`tutorial file` is probably more accurate, and what was there before - I've just moved that code

Reply:
Oh okay, I wondered if it was from hud or something